src/main/java/com/laserfiche/repository/api/RepositoryApiClientImpl.java

```diff
@@ -26,7 +26,7 @@ public class RepositoryApiClientImpl implements RepositoryApiClient, AutoCloseab
     private final TemplateDefinitionsClient templateDefinitionsClient;
 
     protected RepositoryApiClientImpl(String servicePrincipalKey, AccessKey accessKey, String baseUrlDebug) {
-        String baseUrl = baseUrlDebug != null ? baseUrlDebug : "https://api." + accessKey.getDomain() + "/repository";
```

Comment:
I am sorry but I think we moved to using accessors?

Reply:
Oh, Sorry. I fixed it. Thanks.